apis/v1alpha1/httproute_types.go

```diff
@@ -114,6 +131,15 @@ const (
 
 // HTTPRouteMatch defines the predicate used to match requests to a
 // given action.
+// Multiple properties are ANDed together, i.e. the match will evaluate
+// to true if and only if all conditions are satisfied.
```

Comment:
I think the doc can me simplified and still get the message across:

s/if and only if all conditions are satisfied/only if all conditions are satisfied/